API/Repositories/Implementations/PlayerRepository.cs

```diff
@@ -169,7 +169,7 @@ private IQueryable<Player> Search(string username)
             );
         }
 
-        return _context.Players.Where(p => p.Username != null && p.Username.Equals(username, StringComparison.CurrentCultureIgnoreCase));
+        return _context.Players.Where(p => p.Username != null && p.Username.ToLower().Equals(username.ToLower()));
```

Comment:
Refer to my comment in dev chat the other day. LINQ will automatically determine how to compare strings based on the collation of the column that property is mapped to. Prefer usage of just `string.Equals()` and don't worry about messing with case.